src/v/cluster/controller.cc

```diff
@@ -623,7 +671,8 @@ ss::future<> controller::cluster_creation_hook(cluster_discovery& discovery) {
         cmd_data.bootstrap_user_cred
           = security_frontend::get_bootstrap_user_creds_from_env();
         cmd_data.node_ids_by_uuid = std::move(discovery.get_node_ids_by_uuid());
-        co_return co_await create_cluster(std::move(cmd_data));
+        co_await create_cluster(std::move(cmd_data));
+        co_return co_await set_initial_members(discovery.founding_brokers());
```

Comment:
yes, i did, in a current architecture of persisted STM we are only allowed to apply batch to one of the STMs. I think it makes sense as it forces the single responsibility for each of the commands. Do you think that additional raft-0 roundtrip during bootstrap will be a problem. it it should be ususally less than 200ms.

Reply:
Right, I was thinking if we add the brokers to the `create_cluster_cmd` we could have the `bootstrap_backend` take care of this.

I think it shouldn't lead to issues so feel free to leave it as is -- I was unsure whether we wait for Raft membership or members table membership before starting to serve Kafka requests, but it looks like the latter, so additional latency shouldn't open us up to bad behavior afaict.

Reply:
I think the current approach is fine, but having a separate command for initial members instead of adding them to the bootstrap command looks a bit inconsistent to me - we already have initial users and node ids there, why not initial brokers? Or maybe users and node ids should not have been part of the bootstrap command to begin with :)